lib/typescript/httpclient/endpoints/loginViaEmail.ts

```diff
@@ -0,0 +1,24 @@
+import {doFetchFromBackend} from '../api';
+import {User} from '../model';
+import {UserPayload, LoginViaEmailRequestPayload} from '../payload';
+
+const userMapper = (payload: UserPayload): User => {
+  const user: User = {
+    id: payload.id,
+    firstName: payload.first_name,
+    lastName: payload.last_name,
+    displayName: payload.first_name + ' ' + payload.last_name,
+    token: payload.token,
+  };
+  return user;
+};
```

Comment:
Not that I would necessarily change the code here, but I'm wondering if `return {...` would also work (given we're specifying the return type of the function anyway)

Reply:
it does work!